restructuredtext_lint/cli.py

```diff
@@ -1,46 +1,54 @@
 import argparse
 import json
 import sys
+import locale
 
 from .lint import lint_file
 
 
-def _main(filepath, format='text', stream=sys.stdout, encoding=None):
+def _main(filepaths, format, encoding, stream=sys.stdout):
     # Read and lint the file
-    errors = lint_file(filepath, encoding=encoding)
+    file_errors = [lint_file(f, encoding=encoding) for f in filepaths]
+    errors_dicts = []
 
-    # If there were no errors, exit gracefully
-    if not errors:
-        if format == 'json':
-            stream.write('[]')
+    for filename, errors in zip(filepaths, file_errors):
+        if not errors:
+            stream.write('INFO File {} is clean.\n'.format(filename))
         else:
-            stream.write('File was clean.\n')
-        sys.exit(0)
+            if format == 'text':
+                for error in errors:
+                    # e.g. WARNING readme.rst:12 Title underline too short.
+                    stream.write('{} {}:{} {}\n'.format(error.type, error.source, error.line, error.message))
+            elif format == 'json':
+                errors_dicts.extend([{
+                    'line': error.line,
+                    'source': error.source,
+                    'level': error.level,
+                    'type': error.type,
+                    'message': error.message,
+                    'full_message': error.full_message,
+                } for error in errors])
 
-    # Otherwise, output the errors and exit angrily
     if format == 'json':
-        error_dicts = [{
-            'line': error.line,
-            'source': error.source,
-            'level': error.level,
-            'type': error.type,
-            'message': error.message,
-            'full_message': error.full_message,
-        } for error in errors]
-        stream.write(json.dumps(error_dicts))
+        stream.write(json.dumps(errors_dicts))
+
+    if any(file_errors):
```

Comment:
When we relocate everything into a for loop, this can be simplified by adding a boolean marker inside the loop.